app/build.gradle

```diff
@@ -129,25 +130,22 @@ dependencies {
         transitive = true
     }
 
-    // https://mvnrepository.com/artifact/org.apache.commons/commons-math3
-    compile files('libs/commons-math3-3.6.1.jar')
-
     // Butterknife
     compile "com.jakewharton:butterknife:${butterKnifeVer}"
-    apt "com.jakewharton:butterknife-compiler:${butterKnifeVer}"
+    annotationProcessor "com.jakewharton:butterknife-compiler:${butterKnifeVer}"
 
     testCompile "junit:junit:${junitVer}"
     testCompile "org.mockito:mockito-all:${mockitoVer}"
     testCompile "com.squareup.assertj:assertj-android:${androidAssertJVer}"
-    testCompile("org.robolectric:robolectric:${robolectricVer}") {
+    testImplementation("org.robolectric:robolectric:3.5.1") {
         exclude group: 'commons-logging', module: 'commons-logging'
         exclude group: 'org.apache.httpcomponents', module: 'httpclient'
     }
-    testCompile("org.robolectric:shadows-support-v4:${robolectricVer}") {
+    testImplementation("org.robolectric:shadows-support-v4:3.3.2") {
```

Comment:
If you started moving to new android gradle plugin then why not change it everywhere?